static/js/publisher/form/multiselect.js

```diff
@@ -0,0 +1,355 @@
+import React  from 'react';
+import ReactDOM from 'react-dom';
+import PropTypes from 'prop-types';
+
+class MultiSelect extends React.Component {
+  constructor(props) {
+    super(props);
+
+    // hide the original input
+    this.props.input.style.display = 'none';
```

Comment:
Ideally I would say that React components should not touch DOM directly by themselves. Things like that would be better off in some 'init' script that would read the data from input, hide it, and pass the data into React component.

The `init` function down below this class seems like good place to do it.

If it complicates things too much now I'm fine with leaving it for refactor PR which should clear up some things.

Reply:
Moved!